media/css/privacy/cookie-settings-form.scss

```diff
@@ -2,11 +2,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-$font-path: '/media/protocol/fonts';
```

Comment:
I'm fairly certain that we can remove all of these individual `$font-path` and `$image-path` declarations because they will be set only once on the initial load of lib through the base templates. All subsequent "loads" of the lib file should reference this originally loaded file (with config). Need to find a way to test this to confirm.

https://github.com/mozilla/bedrock/blob/main/media/css/protocol/protocol-mozilla.scss#L10
https://github.com/mozilla/bedrock/blob/main/media/css/protocol/protocol-firefox.scss#L9
https://github.com/mozilla/bedrock/blob/main/media/css/protocol/protocol-mozilla-2024.scss#L7

Reply:
For my own future reference, the "privacy" files don't seem to pick up the previous configuration. They use the protocol default `"../img"`

Tested with a wordmark

no config (default path)
<img width="1337" alt="no-config" src="https://github.com/user-attachments/assets/dee5ff58-3579-4276-a1ad-76f3c9949adb" />

with config (expected path)
<img width="1338" alt="with-config" src="https://github.com/user-attachments/assets/87281303-f480-43bd-ac5f-262b7d57d586" />

So it is important to keep these variables when there are internal references within the protocol components (ie. wordmark, logo, modal): https://github.com/search?q=repo%3Amozilla%2Fprotocol%20%24image-path&type=code

Reply:
Yeah, these variables only come into play (if I understand correctly) when a style sheet also loads a protocol component that needs to load either an image or font asset. So things like the modal component, or word marks like you tested with. Otherwise, we should not need to declare them at the top of a file 👍